models/issue_stopwatch.go

```diff
@@ -160,3 +171,31 @@ func SecToTime(duration int64) string {
 
 	return hrs
 }
+
+// APIFormat convert Stopwatch type to api.StopWatch type
+func (sw *Stopwatch) APIFormat() (api.StopWatch, error) {
+	issue, err := getIssueByID(x, sw.IssueID)
+	if err != nil {
+		return api.StopWatch{}, err
+	}
+	return api.StopWatch{
+		Created:    sw.CreatedUnix.AsTime(),
+		IssueIndex: issue.Index,
+	}, nil
+}
+
+// Stopwatches is a List ful of Stopwatch
+type Stopwatches []Stopwatch
```

Comment:
Maybe place it right after `Stopwatch struct`? https://github.com/go-gitea/gitea/blob/973799b4b81d142e5952d517fb41f85894cf0756/models/issue_stopwatch.go#L16

Reply:
done